qabelbox/src/main/java/de/qabel/qabelbox/chat/interactor/TransformingChatUseCase.kt

```diff
@@ -19,14 +20,16 @@ class TransformingChatUseCase @Inject constructor(val identity: Identity, overri
                                                   private val chatServiceUseCase: ChatServiceUseCase) : ChatUseCase {
 
     override fun send(text: String): Observable<ChatMessage> = observable { subscriber ->
-        val item = ChatDropMessage(contact.id, identity.id,
-                ChatDropMessage.Direction.OUTGOING, ChatDropMessage.Status.PENDING,
-                ChatDropMessage.MessageType.BOX_MESSAGE, ChatDropMessage.MessagePayload.TextMessage(text),
-                System.currentTimeMillis())
+        thread {
```

Comment:
instead of thread{}, use the subscribeOn method from Rx:

https://github.com/audax/qabel-android/blob/15933c90331f2e08b56c92dc81ed34ec46fd001a/qabelbox/src/main/java/de/qabel/qabelbox/box/interactor/BoxSharer.kt#L43

That way we don't spawn new threads for every request, but we reuse those from RX